packages/icons/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@littlespoon/icons",
```

Comment:
I forgot to remind you to add icons to `@littlespoon/ui` package

Reply:
I added it. Let me know if I'm missing something.